rules/linux/lnx_system_net_disc_firewall_enum.yml

```diff
@@ -0,0 +1,28 @@
+title: System Network Discovery - Firewall Enumeration
+id: 71da9e5a-fb1e-46a8-abc1-28c80173af4c
+status: experimental
+description: Detects enumeration of firewall configuration
+author: remotephone, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1016/T1016.md
+logsource:
+    category: process_creation
+    product: unix
+detection:
+    selection:
+        CommandLine|contains:
+            # Linux Only
+            - 'arp -a'
+            - 'ip'
+            - 'ss'
+            # macOS and Linux
+            - 'netstat'
+            - 'ifconfig'
```

Comment:
could you please apply the same approach with processname/commandlie as in macos rule?

Reply:
This has been addressed in the latest commit on both rules. Thanks!

Reply:
Thank you!